introduction-to-datasets.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: page
+title: Introduction to Datasets
+---
+
+A dataset is a collection of records, similar to a relational database table. Records are similar to table rows, but the columns can contain not only strings or numbers, but also nested data structures such as lists, maps, and other records.
+
+To define a dataset, Kite minimally requires a [URI](#uris) and a [schema](#schemas).
+
+## URIs
+
+Kite identifies datasets by URI. The URI you provide tells Kite how and where to store data. For example, a dataset created with the URI `dataset:hdfs:/user/cloudera/datasets/movies` is stored in HDFS.
+
+URIs also define a name and namespace for your dataset. Kite uses these values when the underlying system has the same concept (for example, Hive). You can also retrieve the values using the `Datasets.list` method.
+
+Often, the name and namespace provide a location that could be an HDFS path.
+
+To ensure compatibility with underlying systems, names and namespaces in URIs must be mode of alphanumeric characters or underscore (_) and cannot start with a number.
+
+You can create datasets in Hive, HDFS, HBase, or as local files. There are several URI patterns available and additional options.See [Dataset and View URIs]({{site.baseurl}}/URIs.html).
```

Comment:
Missing space between sentences.

Also, I would prefer longer URLs (like URIs.html) as [reference links](http://kramdown.gettalong.org/syntax.html#reference-links) to make it easier to read and maintain the raw markup.